Library/Homebrew/extend/os/cask/artifact/moved.rb

```diff
@@ -0,0 +1,5 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "extend/os/mac/cask/artifact/moved" if OS.mac?
+require "extend/os/linux/cask/artifact/moved" if OS.linux?
```

Comment:
Not something to take on in this PR, but I wonder if we should have a single require file for all of an OS's extensions, rather than introduce three more here. 🤔

Reply:
I think this approach is maybe a bit nicer for now.